src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -802,7 +876,20 @@ protected function processPaymentsAndRefunds(WC_Order $WpObject, int $storekeepe
                 $this->debug('Did not mark the order as paid since it was not needed');
             }
         }
+    }
 
-        return $storekeeper_order;
+    public static function splitStreetNumber(string $streetNumber): array
```

Comment:
is this used? I don't see it returned on the other place

Reply:
Indeed not used, removed it.